spec/models/wishlist_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Wishlist, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
We've got a Rubocop style in place, just for that. You can run check any style error by running `bundle exec rubocop`.